.gitignore

```diff
@@ -1,3 +1,127 @@
-*.pyc
```

Comment:
I appreciate your comment that this kitchen-sink style gitignore is used by other packages, but IMO it shows a lack of control or understanding of your own package. Having git ignore miscellaneous filetypes just serves to hide issues within your own local repo. How does this help?

If there are some unexpected Django (?!) files lurking in the repo, I **want** to see them.

Additionally, the version-control ignore is the wrong place for local setup ignores, i.e. .vscode. Stuff like that belongs in either your system-wide ignore (`~/.config/git/ignore`), or in the repo-wide ignore (`.git/info/exclude`), or in a .gitignore inside the offending folder (`.vscode/.gitignore`).
Imposing your local setup concerns on the public repo seems unnecessary.

Reply:
I resolved this by adding .vscode to .git/info/exclude and removing excess in `.gitignore`.